internal/cli/server_upgrade.go

```diff
@@ -446,6 +448,24 @@ func (c *ServerUpgradeCommand) upgradeRunner(
 					runnerDefaultNames = append(runnerDefaultNames, fmt.Sprintf(runnerDefaultName, cfg.Name))
 					runnerUnsetStr = append(runnerUnsetStr, fmt.Sprintf(runnerUnsetDefault, cfg.PluginType, cfg.Name))
 				}
+
+				// Checking if the default runner profile for a platform, pre-0.9, has the correct task launcher configs
+				// If incorrect, update them
```

Comment:
This comment is (currently) inaccurate, but will leave it until we decide what the upgrade should do here.